mcstats/stats/raids_won.py

```diff
@@ -0,0 +1,12 @@
+from mcstats import mcstats
+
+mcstats.registry.append(
+    mcstats.MinecraftStat(
+        'raids_won',
+        {
+            'title': 'Raider',
```

Comment:
Not sure about this title. More like "Village Defender" or so? You're not the guy raiding, but waiving off a raid.